src/gpl/src/nesterovPlace.cpp

```diff
@@ -241,6 +243,13 @@ void NesterovPlace::init() {
 
   debugPrint(log_, GPL, "replace", 3, "npinit: InitialStepLength {:g}", stepLength_);
 
+  if( pb_->insts().size() < 500 && isnan(stepLength_) ) {
```

Comment:
Please avoid magic numbers. Define a variable such as `const int smallInstCount = 500;` and then use that variable.